libs/remix-ui/search/src/lib/components/Replace.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext, useRef } from 'react'
+import { SearchContext } from '../context/context'
+
+export const Replace = props => {
+  const { setReplace } = useContext(SearchContext)
+  const timeOutId = useRef(null)
+  const change = e => {
+    clearTimeout(timeOutId.current)
+    timeOutId.current = setTimeout(() => setReplace(e.target.value), 500)
```

Comment:
Why is a timeout needed in all components?

Reply:
it's need to delay the input when typing, you don't want to fire searches all the time. would crash...